### PR TITLE
Solution to #228

### PR DESCRIPTION
**Pull request by IamEinstein**
I'm not quite sure, but, most probably this should solve #228.

### CODE REVIEW

**Comment by Haltfire30** [comment]
As I’m understanding this, won’t this only affect both parties not adding pokémon to a trade? Not to mention that some trades occur without any pokémon being added, such as redeem selling or gifting PC. 

There are also slight grammatical issues: “There are no pokemons in this trade!!!”
Should instead read: “There are no pokémon in this trade!”

**Comment by IamEinstein** [comment]
Well ya 😅 
Lemme get back to the code and make the required changes...

**Comment by IamEinstein** [comment]
Ig these changes should do it. But I'm still not very sure as I don't know the exact data structure.

**Comment by EnterXx1** [comment]
E

**Comment by EnterXx1** [comment]
Gib moni

File: cogs/trading.py [hunk 1]
```diff
@@ -355,7 +355,7 @@ async def cancel(self, ctx):
     @checks.has_started()
     @commands.guild_only()
     @trade.command(aliases=("c",))
-    async def confirm(self, ctx):
+    async def confirm(self, ctx, user: discord.Member):
```
Comment (kyle-he) [inline cogs/trading.py:358]: Remove user argument

File: cogs/trading.py [hunk 2]
```diff
@@ -364,8 +364,9 @@ async def confirm(self, ctx):
         if self.bot.trades[ctx.author.id]["executing"]:
             return await ctx.send("The trade is currently loading...")
         
-        if len(self.bot.trades[ctx.author.id]["pokemon"]) <= 0:
-            return await ctx.send("There are no pokemons in this trade!!!")
+        trade = self.bot.trades[user.id]
+        if trade["pokecoins"] <= 0 and trade['redeems'] <= 0 and len(trade['pokemon']) <= 0:
+            return await ctx.send("There are no pokémon/redeems/pokécoins in this trade!")
```
Comment (kyle-he) [inline cogs/trading.py:369]: Replace message with "This trade is empty! Please run `p!trade cancel`" or something more concise/informative.

**Review by kyle-he — CHANGES_REQUESTED** [review]

**Comment by kyle-he** [comment]
Personally, I do not really see this as a bug, as empty trades doesn't really affect either user.

File: cogs/trading.py (hunk 1 above)
Comment (IamEinstein) [reply cogs/trading.py:358]: But if I remove that, then how to get the trade as it requires the `user` argument
```suggestion
     trade = self.bot.trades[user.id]
```

File: cogs/trading.py (hunk 2 above)
Comment (IamEinstein) [reply cogs/trading.py:369]: Ok sure, I'll change that..

**Comment by IamEinstein** [comment]
> Personally, I do not really see this issue as a bug, as empty trades doesn't really affect either user.

Lol, Even I've got the same viewpoint.

**Comment by kyle-he** [comment]
> > Personally, I do not really see this issue as a bug, as empty trades doesn't really affect either user.
> 
> Lol, Even I've got the same viewpoint.

Ok, I'll be closing the PR then

**Pull request closed without merging**